### ISSUE TITLE
Interface Styles plugin has stopped working, needs to be toggled off/on

### ISSUE THREAD

**Issue by dpedu** [runelite/runelite#17060]
### Type

Incorrect behavior

### Operating System

MacOS

### OS Version

11.7.1

### Bug description

I have the Interface Styles plugin enabled and configured as follows: 

<img width="238" alt="image" src="https://github.com/runelite/runelite/assets/1266188/f1ae42be-8403-4953-af6c-45dbf730fd44">

In the last week or so, these styles were not applied when launching the client. They would be applied after toggling the plugin on and off.

### RuneLite version

RL: 1.10.13.1
Launcher: 2.4.2

### Logs

[client.log](https://github.com/runelite/runelite/files/12689708/client.log)
[launcher.log](https://github.com/runelite/runelite/files/12689709/launcher.log)

**Comment by Nightfirecat** [comment]
Seems like it's a bug with the plugin, need to look into it further.
```
2023-09-21 10:31:42 PDT [Client] ERROR n.r.client.callback.ClientThread - Exception in invoke
java.lang.NullPointerException: Cannot read field "am" because "bz.eu" is null
	at client.fi(client.java:40146)
	at client.getWidgetSpriteCache(client.java:44672)
	at net.runelite.client.plugins.interfacestyles.InterfaceStylesPlugin.restoreSprites(InterfaceStylesPlugin.java:218)
	at net.runelite.client.plugins.interfacestyles.InterfaceStylesPlugin.removeGameframe(InterfaceStylesPlugin.java:405)
	at net.runelite.client.plugins.interfacestyles.InterfaceStylesPlugin.updateAllOverrides(InterfaceStylesPlugin.java:164)
	at net.runelite.client.callback.ClientThread.lambda$invoke$0(ClientThread.java:49)
	at net.runelite.client.callback.ClientThread.invokeList(ClientThread.java:119)
	at net.runelite.client.callback.ClientThread.invoke(ClientThread.java:101)
	at net.runelite.client.callback.Hooks.tick(Hooks.java:217)
	at client.ia(client.java:15459)
	at client.bt(client.java)
	at bb.ad(bb.java:358)
	at bb.run(bb.java:335)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by Adam-** [comment]
fixed in 4e8c013af4241b3df64b1c6f6a6ee9d057ecc542
